app/main/views/buyers.py

```diff
@@ -12,26 +12,21 @@
 @role_required('admin', 'admin-ccs-category')
 def find_buyer_by_brief_id():
     brief_id = request.args.get('brief_id')
+    brief = None
+    status_code = 200
 
     try:
-        brief = data_api_client.get_brief(brief_id).get('briefs')
-
+        brief = data_api_client.get_brief(brief_id)
     except:
```

Comment:
We can trim down this try except to a except `HTTPError` looks like that's what we do elsewhere. This is the only instance of such a broad try except and looks like it was probably just an oversight, especially seeing as theres no chance of a `KeyError` now we're not operating on a dictionary (not that there was before).

Reply:
Totally an oversight, will fix.